src/Symfony/Component/Messenger/Bridge/Beanstalkd/Transport/BeanstalkdPriorityStamp.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Bridge\Beanstalkd\Transport;
+
+use Symfony\Component\Messenger\Stamp\StampInterface;
+
+final readonly class BeanstalkdPriorityStamp implements StampInterface
+{
+    public function __construct(
+        public int $priority,
```

Comment:
Maybe?

```suggestion
        public readonly int $priority,
```

Reply:
@fabpot No need, the class itself is readonly.

Reply:
Indeed :)